packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -199,6 +199,7 @@ class BottomSheetCell extends Component {
 				<TouchableOpacity
 					onPress={ onClear }
 					style={ styles.clearIconContainer }
+					testID="clear-button"
```

Comment:
Instead of adding a test ID, I'm wondering if we could set accessibility properties to this component, as we do for the main button, in fact, this would help us improve the accessibility of the editor. Wdyt?

https://github.com/WordPress/gutenberg/blob/666e9ea5d18eed5a337c0d21faa7b356bfa4dce0/packages/components/src/mobile/bottom-sheet/cell.native.js#L355-L368

Reply:
Good callout! I've update the test and accessibility labels here. The one part I am curious on is if I have handled the i18n effectively here. Tracing back the thread to the [i18n __( ... )](https://github.com/WordPress/gutenberg/blob/52bac1c1d524b333c19872a07854d7e7121dc337/packages/i18n/README.md#__) method, the documentation suggests this is ok to just use a text string here. Let me know if this needs further investigation.

Reply:
> Tracing back the thread to the i18n __( ... ) method, the documentation suggests this is ok to just use a text string here. Let me know if this needs further investigation.

Yeah, the usage of the i18n function is right. We don't need to take further actions regarding translations, as this is automatically handled by the translation pipeline.